requirements.txt

```diff
@@ -1,8 +1,10 @@
 batinfo==0.3
+-e git+https://github.com/bochecha/dbus-python.git@pipified-1.1.1#egg=dbus
```

Comment:
Not sure why we are not using upstream fork instead; https://github.com/posborne/dbus-python/network graph is not clear. Can you elaborate a bit?
Also, it would be great to push it on pypi. Have you been in contact with the upstream fork maintainers about that?
(Also, no need to -e in this case :) )